src/module-elasticsuite-core/Search/Adapter/Elasticsuite/Request/Query/Builder/QueryMatch.php

```diff
@@ -24,7 +24,7 @@
  * @package  Smile\ElasticsuiteCore
  * @author   Aurelien FOUCRET <aurelien.foucret@smile.fr>
  */
-class Match implements BuilderInterface
+class QueryMatch implements BuilderInterface
```

Comment:
Can you name it "MatchQuery" instead of "QueryMatch" ?

Reply:
Done.